src/bson/ext/uint.cr

```diff
@@ -0,0 +1,47 @@
+struct UInt8
```

Comment:
Same for this whole file

Reply:
Fixed in following commit : 988b1c24c7e520fec575478e0ba307c40b779a3e